gatsby/src/components/__tests__/hello.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react"
```

Comment:
muze bit drzet test soubory spoly s originalni? treba
```
components
  hello.tsx
  hello.scss
  hello.xtest.tsx
```

Reply:
Jasně, určitě to jde, akorát se na tom musíme dohodnout v týmu. V tomto případě by to bylo `index.test.tsx`.

Reply:
cc @vasimi @breberafilip

Reply:
Za mě držet spolu a komponenty balit do složek